src/components/UserInfo/UserInfo.tsx

```diff
@@ -2,0 +1,25 @@
+import React from 'react';
+import { Todo, User } from '../../types/types';
+import usersFromServer from '../../api/users';
+
+type Props = {
+  todo: Todo;
+};
+
+function getUser(arrUsers: User[], userId: number): User | undefined {
+  return arrUsers.find(el => el.id === userId);
+}
+
+export const UserInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUser(usersFromServer, todo.userId);
```

Comment:
Instead of fetching users here, consider passing the user data or the specific user object as a prop from the `TodoList` or `App` component to maintain a single source of truth for data.